ros2bag/ros2bag/api/__init__.py

```diff
@@ -148,6 +148,14 @@ def add_standard_reader_args(parser: ArgumentParser) -> None:
              'By default attempts to detect automatically - use this argument to override.')
 
 
+def add_standard_node_args(parser: ArgumentParser, prefix: str) -> None:
+    """Add arguments for for verbs that create a Node."""
```

Comment:
```suggestion
    """Add arguments for verbs that create a Node."""
```
Nitpick, need to remove extra `"for"`.